stories/generic-tile/generic-tile.stories.js

```diff
@@ -80,149 +80,1289 @@ genericTileSizes.parameters = {
             <div class="fd-tile__title">.fd-tile__title</div>
             <div class="fd-tile__subtitle">.fd-tile__subtitle</div>
         </div>
-        <div class="fd-tile__content fddocs-tile-example-border">.fd-tile__content</div>
-        <div class="fd-tile__footer fddocs-tile-example-border">.fd-tile__footer</div>
+        <div class="fd-tile__content" style="background-color: #C0E0FA">.fd-tile__content</div>
+        <div class="fd-tile__footer" style="background-color: #C0E0FA">.fd-tile__footer</div>
     </div>
 </div>
-            `;
+`;
 
-Layout.storyName = 'Layout';
+genericTileLayout.storyName = 'Layout';
 
-Layout.parameters = {
+genericTileLayout.parameters = {
     docs: {
         iframeHeight: 500,
         storyDescription: `
-The generic tile can be displayed in two tile dimensions: the default 1x1, and 2x1 by adding the \`fd-tile--double\` modifier class.
+The layout of the generic tile is fixed with designated areas for the header, content area and footer. 
 
-**Generic tiles are typically displayed with the following dimensions:**
-        
-- Large tile (1 x 1) - 11 x 11rem
-- Large tile (2 x 1) - 22.5 x 11rem
-- Small tile (1 x 1) - 9.25 x 9.25rem
-- Small tile (2 x 1) - 19 x 9.25rem
-        
-
-The layout of the generic tile is fixed with designated areas for the header, content area, and footer. Below are the elements typically found in generic tiles:
+**The common elements displayed in generic tiles are:**
 
-Element | Modifier class | Description
-:-------- | :-------------------- | :---------------
+Element | Class | Description
+:------ | :------------- | :-----------
 Header (mandatory) | \`fd-tile__header\` | The tile displays a title and an optional subtitle.
 Title (mandatory) | \`fd-tile__title\` | The title can display up to two lines of text before it truncates (or three lines if there is no subtitle).
 Subtitle (optional) | \`fd-tile__subtitle\` | The subtitle can display one line of text before it truncates.
 Content (optional) | \`fd-tile__content\` | Generic tile can display content if necessary.
 Footer (optional) | \`fd-tile__footer\` | The footer can contain a refresh icon \`sap-icon--refresh\` and/or status.
 
-Note: the background color displayed in the example below is only for illustrative purposes.
+**Note:** In the example below, the background color (and class names) illustrate the placement of elements within tiles.
         ` }
 };
 
-export const Columns = () => `<div class="fddocs-container">
+export const genericTileContentColumns = () => `<div class="fddocs-container">
     <div role="button" aria-label="button" tabindex="0" class="fd-tile">
         <div class="fd-tile__header fd-tile__header--2-col">
-            <div class="fd-tile__section fddocs-tile-example-border"></div>
-            <div class="fd-tile__section fddocs-tile-example-border"></div>
+            <div class="fd-tile__section" style="background-color: #C0E0FA"></div>
```

Comment:
@jessmetivier is this intentional?

Reply:
@droshev I have no idea when/how that happened actually. @InnaAtanasova do you have any idea? All I did was change the documentation and added the "specific tile" code to "generic tile" so that they are all together.

Reply:
Now, when I go switch to my branch... I'm seeing this:
<img width="773" alt="Screen Shot 2020-11-18 at 9 43 11 AM" src="https://user-images.githubusercontent.com/57453945/99544617-83e14d00-2982-11eb-8d73-c105558d05bf.png">

Reply:
@jessmetivier I am taking care of it :) thanks for the work